share/smack/top.py

```diff
@@ -197,6 +197,9 @@ def arguments():
   translate_group.add_argument('--integer-overflow', action='store_true', default=False,
     help='enable integer overflow checks')
 
+  translate_group.add_argument('--llvm-assumes', choices=['none', 'use', 'check'], default='none',
+    help='optionally enable generation of Boogie assume statements from LLVM assume statements (none=no generation [default], use=generate assume statements, check=check validity of assume statemets)')
```

Comment:
You have a typo here. Also, change the last option into `check assume statements`.

Reply:
Done.